src/java/org/apache/sqoop/metastore/GenericJobStorage.java

```diff
@@ -675,6 +736,14 @@ private void initV0Schema() throws SQLException {
     }
   }
 
+  private void checkForOldSchema() throws SQLException {
```

Comment:
I would call this checkForOldRootProperties or something because the schema itself did not change just the ROOT table and a property in it, right?